### ISSUE TITLE
Button or checkbox to expose all the plugins parameters

### ISSUE THREAD

**Issue by crshrprt** [BespokeSynth/BespokeSynth#1691]
## Summary

Right now plugin's parameters are exposed only when wiggled. For UI usability make sense to hide parameters when they exceed a certain number, some plugins can have hundreds of parameters. However, I usually use plugins with less than 10 parameters (like many Airwindows ones) and sometimes some other plugins that have 20 or 30 parameters max.

Would be useful to have a button on the plugin module itself, or in its triangle menu (as a checkbox?), to expose all plugin parameters in one single operation.

### Motivation

Right now I save an "init" preset for my most used plugins just to have a quick way to expose all the plugins parameters in BespokeSynth, but for plugins with, let's say, more than 10 parameters is going to take minutes to wiggle all the parameters and make sure they go back to their init state, for example. It would make it much quicker to have a feature to expose all plugins parameters at once.

**Comment by KottV** [comment]
Good point, especially some plugins and formats (LV2) don't work well with wiggling. But it should be limited anyway - not all at once, because some synths have thousands of params, so I think it may has a paged approach like "expose up to 32 next parameters".

**Comment by seastones** [comment]
Posting a suggestion I gave in the Discord: It may be beneficial to also have a way to hide parameters after they've been exposed.
The idea in my mind is that it would allow a user to combat the potential clutter caused by exposing a large number of (potentially) unused parameters.
